java-base/Dockerfile

```diff
@@ -17,13 +17,19 @@ LABEL name="Stackable image for OpenJDK" \
     summary="The Stackable OpenJDK base image." \
     description="This image is the base image for all Stackable Java product images."
 
-# We need to use EPEL, as openjdk 22 is not shipped with UBI9
-RUN rpm --install --replacepkgs https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+RUN cat <<EOF > /etc/yum.repos.d/adoptium.repo
+[Adoptium]
+name=Adoptium
+baseurl=https://packages.adoptium.net/artifactory/rpm/rhel/\$releasever/\$basearch
```

Comment:
Re Acceptance Criteria: _Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there_

@Jimvin @lfrancke, Can we pull this through Nexus?

Reply:
Not yet, we have a mirror but not everything is in place yet.
This is okay for now.